server/models/role.ts

```diff
@@ -53,24 +53,24 @@ export default (
     underscored: true,
     indexes: [
       { fields: ['address_id'] },
-      { fields: ['offchain_community_id'] },
+      // { fields: ['offchain_community_id'] },
       { fields: ['chain_id'] },
       { fields: ['address_id', 'chain_id'], unique: true },
-      { fields: ['address_id', 'offchain_community_id'], unique: true },
+      // { fields: ['address_id', 'offchain_community_id'], unique: true },
     ],
     validate: {
       // roles should only have 1 of these properties
-      eitherOffchainOrOnchain() {
-        if (this.chain_id && this.offchain_community_id) {
-          throw new Error('Roles can have either chain_id or offchain_community_id, not both!');
-        }
-      }
+      // eitherOffchainOrOnchain() {
```

Comment:
We should go through and ensure that chain_id is no longer allowed to be null (on basically all server changes above). That will act as a stronger form of validation across the board and may need to be written into your migration.

Reply:
I'm adding new column constraints as part of the migration. Only exception is Subscriptions, as they're going through a redesign soon.

Reply:
e.g. https://github.com/hicommonwealth/commonwealth/pull/1585/files#diff-e141ba737f645d8a7158fa446cd93172bdbc5e82cd04991d43aef5b701c45b64R79